controllers/errors/views/error.njk

```diff
@@ -1,39 +1,33 @@
-{% from "components/hero.njk" import hero with context %}
-
 {% extends "layouts/main.njk" %}
+{% from "components/content-box/macro.njk" import contentBox %}
+{% from "components/hero.njk" import hero with context %}
 
 {% block content %}
     <main role="main">
-        {{ hero(
-            titleText = title,
-            image = fallbackHeroImage,
-            accent = pageAccent
-        ) }}
+        {{ hero(title, fallbackHeroImage) }}
 
         <div class="nudge-up">
-            <div class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
-                <div class="s-prose">
-                    <h3 class="t2">Error {{ status }}</h3>
+            {% call contentBox() %}
```

Comment:
This file looks churn-y but it's only wrapping the content in a `contentBox` macro.